crypto/src/main/java/net/corda/v5/crypto/KeyUtils.java

```diff
@@ -75,18 +74,15 @@ public static boolean isKeyInSet(@NotNull PublicKey key, @NotNull Iterable<Publi
      * check fulfilment against a set of keys, without having to handle simple and composite keys separately (that is, this is
      * polymorphic).
      * 
-     * @param firstKey  The key with the requirements.
+     * @param key  The key with the requirements.
      * @param otherKeys The key to check whether requirements are fulfilled.
      */
-    public static boolean isKeyFulfilledBy(@NotNull PublicKey firstKey, @NotNull Iterable<PublicKey> otherKeys) {
```

Comment:
I would expect to see https://github.com/corda/corda-api/blob/kyriakos/api-refinements/crypto/src/main/java/net/corda/v5/crypto/CompositeKey.java#L49 signature changed as well

Reply:
Good catch, will apply